dropshot_endpoint/src/lib.rs

```diff
@@ -41,6 +41,8 @@ mod util;
 ///     deprecated = { true | false },
 ///     // A value of `true` causes the operation to be omitted from the API description
 ///     unpublished = { true | false },
+///     // Optional maximum request body size in bytes
+///     request_body_max_bytes = 1 * 1024 * 1024,
```

Comment:
I know `tags` indicates that it's optional, but all these are optional. I would suggest:

```
/// #[endpoint {
///     // Required fields
///     method = { DELETE | HEAD | GET | OPTIONS | PATCH | POST | PUT },
///     path = "/path/name/with/{named}/{variables}",
///
///     // Optional fields
///     // Tags to be included in the OpenAPI document
///     tags = [ "all", "your", "OpenAPI", "tags" ],
///     // Specifies the media type used to encode the request body
///     content_type = { "application/json" | "application/x-www-form-urlencoded" | "multipart/form-data" }
///     // A value of `true` marks the operation as deprecated
///     deprecated = { true | false },
///     // A value of `true` causes the operation to be omitted from the API description
///     unpublished = { true | false },
///     // Maximum request body size in bytes; may be any rust expression including a constant.
///     request_body_max_bytes = 1 * 1024 * 1024,
```

Small suggestion for clarity of the value, but I'm not in love with what I wrote...

Reply:
I'll try and make this clearer, good catch.